### ISSUE TITLE
Skip Weekends

### ISSUE THREAD

**Issue by WaltDaniels** [gavynriebau/Dates.Recurring#10]
First of all, awesome library.  I've been asked to implement Outlook-style recurring calendar events into a project and Dates.Recurring can do all of the permutations required except one.  We need events to occur every "x" days (which you support) with the option to include or exclude weekends.  I started looking at your "Daily.cs" file with the intent of either modifying it or cloning it and creating a "WeekDay.cs" option.  Does this sound like a reasonable approach?  Or, do you have another suggestion to support the "exclude weekends" feature that I need?

**Comment by gavynriebau** [comment]
@WaltDaniels I think extending the Daily.cs file would work well. You'll also want to add a builder method to DaysBuilder.cs.

**Comment by gavynriebau** [comment]
@WaltDaniels thanks, great to hear the feedback.

I think extending Daily.cs would be a good approach with a corresponding new builder method in DaysBuilder.cs.

Rather than just wholesale excluding weekend days it would also be worth allowing any day(s) of the week to be excluded . That way if anyone else wants to exclude a particular day they also could do so.
You can take a look at the "OnDays" method in WeeksBuilder.cs for an example of what I mean.

You would still be able to exclude weekend days using `Day.SATURDAY | Day.SUNDAY`
e.g.
```csharp
var daily = Recurs
    .Starting(new DateTime(2015, 1, 1))
    .Every(1)
    .Days()
    .Excluding(Day.SATURDAY | Day.SUNDAY)
    .Ending(new DateTime(2015, 1, 15))
    .Build();

daily.Next(new DateTime(2014, 7, 3));  // 2015/01/01
daily.Next(new DateTime(2015, 1, 1));  // 2015/01/02
daily.Next(new DateTime(2015, 1, 2));  // 2015/01/05
daily.Next(new DateTime(2015, 1, 15)); // null
```

**Comment by gavynriebau** [comment]
@WaltDaniels thanks, great to hear the feedback.

I think extending Daily.cs would be a good approach with a corresponding new builder method in DaysBuilder.cs.

Rather than just wholesale excluding weekend days it would also be worth allowing any day(s) of the week to be excluded . That way if anyone else wants to exclude a particular day they also could do so.
You can take a look at the "OnDays" method in WeeksBuilder.cs for an example of what I mean.

You would still be able to exclude weekend days using `Day.SATURDAY | Day.SUNDAY`
e.g.
```csharp
var daily = Recurs
    .Starting(new DateTime(2015, 1, 1))
    .Every(1)
    .Days()
    .Excluding(Day.SATURDAY | Day.SUNDAY)
    .Ending(new DateTime(2015, 1, 15))
    .Build();

daily.Next(new DateTime(2014, 7, 3));  // 2015/01/01
daily.Next(new DateTime(2015, 1, 1));  // 2015/01/02
daily.Next(new DateTime(2015, 1, 2));  // 2015/01/05
daily.Next(new DateTime(2015, 1, 15)); // null
```

**Comment by WaltDaniels** [comment]
I created a pull request with this new feature.  I implemented a .Including() method to selectively add the days to include in the pattern.

**Comment by gavynriebau** [comment]
Thanks @WaltDaniels really nice work here.
Resolved in pull request #11 and package version 1.2.28
